src/routes/(console)/+layout.svelte

```diff
@@ -259,7 +262,36 @@
             rank: -1
         }
     ]);
+
+    function initReo() {
+        if (dev || !browser || !isCloud) return;
+
+        const clientID = '144fa7eaa4904e8';
+        const reoPromise = loadReoScript({ clientID });
+
+        reoPromise.then(async (reo: Reo) => {
+            reo.init({ clientID });
+
+            if (Object.keys($user).length) {
+                const name = $user.name || $user.email;
+                const nameParts = name.trim().split(' ');
+                const lastname = nameParts.length > 1 ? nameParts.slice(1).join(' ') : undefined;
+
+                const reoIdentity: ReoUserIdentifyConfig = {
+                    username: $user.email,
+                    firstname: nameParts[0],
+                    type: $user.password ? 'email' : 'github',
+                    ...(lastname && { lastname })
+                };
+
+                reo.identify(reoIdentity);
+            }
+        });
+    }
+
     onMount(async () => {
+        initReo();
```

Comment:
So this will only identify the user if they're already logged in. Typically, when doing analytics, you also need to identify the user when they sign in as well. Otherwise, we won't get anything for people who sign up, play around, and never come back.

Reply:
Updated.